src/routes/simulateur/Forms/NumberForm.tsx

```diff
@@ -0,0 +1,49 @@
+import { component$, Signal, useComputed$, QRL, $ } from "@qwik.dev/core";
+import { steps } from "../steps";
+import { Answer } from "..";
+
+interface NumberProps {
+  add: QRL<(answer: Answer, nextOption?: string) => any>;
+  current: Signal<string>;
+}
+
+export default component$<NumberProps>(({ current, add }) => {
+  const step = useComputed$(() => steps[current.value]);
+
+  const onSubmit = $((form: HTMLFormElement) => {
+    if (step.value.type !== 'number') return;
+    const formData = new FormData(form);
+    const formValue: Record<string, string> = {}
+    for (const [key, value] of formData.entries()) {
+      const label = step.value.props.fields[key].label;
+      formValue[key] = `${label}: ${value}`;
+    }
+    const answer: Answer = {
+      question: current.value,
+      option: '',
```

Comment:
rename "response" everywhere:
```suggestion
      response: formValue,
```

Reply:
done